reports/usa.json

```diff
@@ -190,7 +190,11 @@
       "query": {
         "dimensions": ["ga:pageTitle", "ga:eventLabel", "ga:pagePath"],
         "metrics": ["ga:totalEvents"],
-        "filters": ["ga:eventCategory=~ownload", "ga:eventLabel!~swf$"],
+        "filters": [
+          "ga:eventCategory=~ownload", 
+          "ga:eventLabel!~swf$",
+          "ga:pagePath!~.*\/(zip|doc).*"
+        ],
```

Comment:
As it stands, I think this is too broad. We don't want to exclude every URL with `/documents/` in it, for example. Since this is a patch -- what do you think of limiting it to `usps.com` URLs, since that's the only thing that motivated this patch?